deploy.sh

```diff
@@ -1,6 +1,6 @@
 set -e
 
-API="https://api.cloud.gov"
+API="https://api.fr.cloud.gov"
```

Comment:
Hm, this is the cloud.gov _API_ url, not the application url. 

Regarding the app url, we're not explicitly defining the fr.cloud.gov url, so maybe that was added by default? I'm content to remove it. I don't see a reason to drop the app.cloud.gov binding, though?